curriculum/js1/6.js

```diff
@@ -5,8 +5,11 @@
  * @returns {number}
  */
 
-const solution = (a) => {
-  return 0;
+const solution = (a, b=a, result=0) => {
+  if (b === 0) return result;
+  if ((b !== 1) && (a !== b) && (a % b === 0)) result += b;
```

Comment:
Unnecessary parenthesis due to precedence of logical operators: https://chortle.ccsu.edu/java5/Notes/chap40/ch40_16.html